app/src/main/java/io/euphoria/xkcd/app/impl/connection/ConnectionManagerImpl.java

```diff
@@ -35,7 +34,7 @@ public synchronized Connection getConnection(String roomName) {
     public synchronized Connection connect(String roomName) {
         ConnectionImpl conn = connections.get(roomName);
         if (conn == null) {
-            conn = new ConnectionImpl(this, roomName, settings.shouldContinuePrevSession() ? settings.getSessionCookie() : null);
+            conn = new ConnectionImpl(this, roomName);
```

Comment:
Looks like the `shouldCountinuePrevSession` check went under during the refactoring; I'll re-add it before merging.

Reply:
Resolved in 8ffdc1c84a6c2dcee6f8336333a343def58ffa09.